src/lib/containers/table/SynapseTable.tsx

```diff
@@ -218,7 +228,12 @@ export default class SynapseTable extends React.Component<
         this.resizer = new ColumnResizer(this.tableElement, RESIZER_OPTIONS)
       }
     } else {
-      this.resizer.reset(RESIZER_OPTIONS)
+      // TODO: use event driven solution instead of timeout.
```

Comment:
Initially the column resizer grip positions were... off.  I think it's due to the SynapseTableCell refactor (which I totally support!), but I'm sure there's a better way than this (so I left a note)